### PR TITLE
Remove headline ab test

### PR DESCRIPTION
**Pull request by dominickendrick**
## What does this change?

Removes headline AB test due to lack of uptake 😢 
## What is the value of this and can you measure success?

Deletes code !
## Does this affect other platforms - Amp, Apps, etc?

Fastly https://github.com/guardian/fastly-edge-cache/pull/666
## Request for comment

@TBonnin

### CODE REVIEW

File: common/app/mvt/MultiVariateTesting.scala [hunk 1]
```diff
@@ -16,6 +16,8 @@ import conf.switches.Switches.ServerSideTests
 //    val tests = List(ExampleTest)
 // }
 
+<<<<<<< HEAD
+=======
```
Comment (TBonnin) [inline common/app/mvt/MultiVariateTesting.scala]: 😊

File: common/app/headlines/ABHeadlines.scala [hunk 2]
```diff
@@ -1,110 +0,0 @@
-package headlines
-
-import app.LifecycleComponent
-import com.google.gdata.client.spreadsheet.{FeedURLFactory, SpreadsheetService}
-import com.google.gdata.data.spreadsheet.{CellFeed, ListFeed, WorksheetFeed}
```
Comment (TBonnin) [inline common/app/headlines/ABHeadlines.scala]: Which lib are those spreadsheet dependencies coming? Is it used anywhere else?

Comment (dominickendrick) [inline common/app/headlines/ABHeadlines.scala]: Good point!

**Comment by TBonnin** [comment]
👍

**Pull request merged**